### PR TITLE
[#158755843] Ratings And Reviews

### PR DESCRIPTION
**Pull request by clintfidel**
#### What does this PR do?
This PR allows allows users review and rate a particular product in the store
#### Description of Task to be completed?
- add model for storing users reviews and ratins
- create plugin for reviews and rates
- add forms to add reviews and rates
#### How should this be manually tested?
- pull the branch
- type reaction to start the app
- sign in as a user, select a product
- At the rare right enter a review and then rate also
- Click on Add Review burron
#### Screenshots (if appropriate)
screen shot 2018-07-19 at 12 41 26 pm
screen shot 2018-07-19 at 1 19 58 pm

#### What are the relevant pivotal tracker stories?
[Finishes #158755843](https://www.pivotaltracker.com/story/show/158755843)

#### Any background context you want to provide?
N/A

### CODE REVIEW

**Comment by saheedt** [comment]
We could improve on the error message grammar. 
A suggestion is 👉🏽 'your rating is needed to serve you better'

File: .eslintrc [hunk 1]
```diff
@@ -14,7 +14,9 @@
     "Alerts": true,  // this needs to get cleaned up, but ignoring for now
     "jest": true,
     "expect": true,
-    "test": true
+    "test": true,
+    "toastr": true
```
Comment (Daramola98) [inline .eslintrc]: Good job @clintfidel please kindly uninstall the toastr package since we are not making use of it anymore

Comment (Daramola98) [reply .eslintrc]: I cannot seem to find any file relating to the code written for this feature in the files changed section @clintfidel

**Comment by Daramola98** [comment]
@clintfidel Check the project wiki for the correct branch naming convention

**Pull request closed without merging**